scripts/linters/js_ts_linter.py

```diff
@@ -429,6 +433,74 @@ def _check_angular_services_index(self) -> concurrent_task_utils.TaskResult:
         return concurrent_task_utils.TaskResult(
             name, failed, error_messages, error_messages)
 
+    def _check_unknown_type(self) -> concurrent_task_utils.TaskResult:
+        """Prints a list of lint errors if an unknown type is used. This lint
+            check is not enabled by default. Add proper comment if unknown is
+            needed.
```

Comment:
This docstring is not correctly formatted. Please see our style guide for instructions. In particular, the second and third lines should not be indented.